server/controllers/schedule.js

```diff
@@ -8,10 +8,11 @@
 var secrets = require('../config/secrets');
 var form = require('../models/form');
 var Github = require('../models/github');
-var github = new Github(
-  secrets.github.clientID,
-  secrets.github.clientSecret
-);
+var github = new Github({
+  client: secrets.github.clientID,
+  secret: secrets.github.clientSecret,
+  token: secrets.github.token
```

Comment:
The reason this worked is that the token is optional (it just avoids rate limiting).